{{ cookiecutter.project_name }}/Dockerfile

```diff
@@ -18,14 +18,14 @@ RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
 
 WORKDIR /app
 
-# Pin a known to work with peep pip version.
-RUN pip install pip==6.0.0
+# Get pip8
+COPY bin/pipstrap.py bin/pipstrap.py
```

Comment:
Hehe I get the confusion, here's what's happening:
 - Docker starts building an image from the base defined in `FROM`. The image has nothing else from your code yet
 - You add files into your image with `ADD` and `COPY`.
 - Here we're copying `bin/pipstrap.py` from the source of the host, i.e. the directory you're working on into the docker image.
 - In the next line we run `pipstrap` to get `pip8`